src/qt/forms/coincontroldialog.ui

```diff
@@ -375,6 +375,13 @@
           </property>
          </widget>
         </item>
+        <item>
+         <widget class="QLabel" name="labelSelected">
+          <property name="text">
+           <string notr="true">(1 selected)</string>
```

Comment:
Incorrect?

Reply:
???,
placeholder string, as in labelLocked

Reply:
Don't know why that is there too, and I think there is no special reason. I guess it's ok since it is consistent with others.